libraw/src/main/java/com/homesoft/photo/libraw/LibRaw.java

```diff
@@ -36,6 +39,34 @@ public class LibRaw implements AutoCloseable {
 
     long mNativeContext;
 
+    public static class CameraWhiteBalance {
```

Comment:
this could be a record class but desugaring is often annoying to deal with and I don't want to force it.